src/Subscriber/FindReviewerSubscriber.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Api\PullRequest\PullRequestApi;
+use App\Command\SuggestReviewerCommand;
+use App\Entity\Task;
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use App\Service\TaskScheduler;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class FindReviewerSubscriber implements EventSubscriberInterface
+{
+    private $pullRequestApi;
+    private $botUsername;
+    private $scheduler;
+
+    public function __construct(PullRequestApi $pullRequestApi, TaskScheduler $scheduler, string $botUsername)
+    {
+        $this->pullRequestApi = $pullRequestApi;
+        $this->botUsername = $botUsername;
+        $this->scheduler = $scheduler;
+    }
+
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        if (!in_array($data['action'], ['opened', 'ready_for_review']) || ($data['pull_request']['draft'] ?? false)) {
+            return;
+        }
+
+        $repository = $event->getRepository();
+
+        // set scheduled task to run in 20 hours
+        $this->scheduler->runLater($repository, $data['number'], Task::ACTION_SUGGEST_REVIEWER, new \DateTimeImmutable('+20hours'));
```

Comment:
Ok, so when a new PR is opened, or "ready_for_review", then, 20 hours later, we will run the "suggest reviewer" functionality. 20 hours seems fairly fast to me maybe ?

Reply:
I also wonder what to do with PRs opened with the "Status: Needs Work" label for whatever reasons (or changed with Carson bot). But that's probably the same as opening a Draft PR, we don't want it?

Reply:
the status is set to `needs work` when a reviewer asks for changes. Saying we don't want such PRs would be an issue. It means we only accept PRs when they are perfect at first review or when the contributor can change the PR and re-request a review in less than 20 hours after receiving a review (which is putting a lot of burden on them when they contribute in their free time)

Reply:
@weaverryan Maybe it is too fast, maybe not. Whenever I make a PR I am secretly and eagerly waiting for some kind of feedback. So 20 hours is suuuper long. 

@ogizanagi, I've not been thinking of that.. I should make sure the PR has the correct status. 

A user may open a PR, then change the status them selves. If so, we should not call for a review.

Reply:
I think everyone likes to get a review/comment within a couple hours after opening. However, there is a difference between what we think is reasonable and what we like. 

If I think "when does someone start to add a ping or asks on Slack if someone can please take a look at their PR", I think this margin is closer to 5 days - 1 week than to 20 hours.

Reply:
Should we try with 20 hours for the first 10 pings? Just to see what happens. 

Also note that none of us never will be pinged. We will mostly ping symfony contributors that nobody ever pinged directly on github or slack.

Reply:
FYI: The rust community have a bot that pings potential reviewers as soon as the PR is open. 

```
https://github.com/rust-lang/rust/pull/78714
```